facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -200,6 +214,8 @@ private static void printUsageAndExit() {
         System.out.println();
         System.out.println("To disable sound use the " + NO_SOUND + " launch argument (default in headless mode).");
         System.out.println();
+        System.out.println("To change the port the server is hosted on use the " + SERVER_PORT + " launch argument.");
```

Comment:
I know little about how Terasology handles networking, but may I recommend to add a "(default: xxxxx)"  at the end of this help string, where xxxxx is the default port?

Also, could we add an example for this or expand one of the existing ones? Perhaps the headless example could be expanded for this purpose.